nwbwidgets/controllers/group_and_sort_controllers.py

```diff
@@ -201,7 +201,7 @@ def set_group_by(self, group_by):
             groups = np.unique(keep_column_values)
             self.group_sm.rows = min(len(groups), 20)
             self.group_sm.options = tuple(groups[::-1])
-            self.group_sm.value = self.group_sm.options
+            self.group_sm.value = self.group_sm.options[:1]
```

Comment:
What is this doing?

Reply:
In my attempts to investigate why plotting is slow when I change the values of group and sort, I came across one of these possibilities: after selecting a col by which to group by, the  select multiple drop down widget defaults to all group values at the start, which then cascades down to the plot adding all of those traces. So I think it might be better to default to one value from the drop down and the user can explicitly select multiple if needed. Check the image below (it defaults to 12 unlike before where it default to all values): 
![image](https://user-images.githubusercontent.com/11567561/122145153-5609ab80-ce72-11eb-9d15-9d5ed143aec6.png)

Reply:
Makes sense. We could even select the empty set to start

Reply:
yeah thats even better actually! changed it https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets/pull/169/commits/0e21776323d42b95b7919a37caee4b773c1203bb